### PR TITLE
empty lines removed, UserDTO with kingdom created, service and contro…

### PR DESCRIPTION
**Pull request by bleaksmile**
…ller refactored

### CODE REVIEW

**Review by ChrisFrill — APPROVED** [review]
Thanks Eni! I approve.

File: src/main/java/com/greenfox/tribes1/Security/WebSecurity.java [hunk 1]
```diff
@@ -22,14 +22,12 @@ public WebSecurity(UserDetailsServiceImpl userDetailsService) {
   @Override
   public void configure(HttpSecurity http) throws Exception {
     http.csrf().disable().authorizeRequests()
-            .antMatchers( "/register", "/login").permitAll()
+            .antMatchers( "/register", "/login", "/test", "/heroku").permitAll()
```
Comment (csadesz) [inline src/main/java/com/greenfox/tribes1/Security/WebSecurity.java]: In this situation, you can easily use "/**"

File: src/main/java/com/greenfox/tribes1/ApplicationUser/DTO/ApplicationUserDTO.java [hunk 2]
```diff
@@ -0,0 +1,21 @@
+package com.greenfox.tribes1.ApplicationUser.DTO;
+
+import lombok.*;
```
Comment (csadesz) [inline src/main/java/com/greenfox/tribes1/ApplicationUser/DTO/ApplicationUserDTO.java]: Try to not import all the elements of the library the necessary ones

**Review by csadesz — CHANGES_REQUESTED** [review]

File: src/main/java/com/greenfox/tribes1/Security/WebSecurity.java (hunk 1 above)
Comment (bleaksmile) [reply src/main/java/com/greenfox/tribes1/Security/WebSecurity.java]: Not wanted to add permit to all endpoints with /** in oder to not mess the code up later on ...

File: src/main/java/com/greenfox/tribes1/ApplicationUser/DTO/ApplicationUserDTO.java (hunk 2 above)
Comment (bleaksmile) [reply src/main/java/com/greenfox/tribes1/ApplicationUser/DTO/ApplicationUserDTO.java]: can not resolve this issue now as the current version of the program is not running on my pc .... but will do so!

File: src/main/java/com/greenfox/tribes1/Security/WebSecurity.java (hunk 1 above)
Comment (bleaksmile) [reply src/main/java/com/greenfox/tribes1/Security/WebSecurity.java]: corrected as per requested

File: src/main/java/com/greenfox/tribes1/ApplicationUser/DTO/ApplicationUserDTO.java (hunk 2 above)
Comment (bleaksmile) [reply src/main/java/com/greenfox/tribes1/ApplicationUser/DTO/ApplicationUserDTO.java]: corrected as per requested

**Review by csadesz — APPROVED** [review]

**Pull request merged**